app/models/manageiq/providers/container_manager/container_template.rb

```diff
@@ -0,0 +1,67 @@
+class ManageIQ::Providers::ContainerManager::ContainerTemplate < ContainerTemplate
```

Comment:
If this is Kubernetes specific code, why is it going in ManageIQ::Providers::ContainerManager::ContainerTemplate ?